src/Sylius/Bundle/CoreBundle/Tests/DependencyInjection/SyliusCoreExtensionTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Tests\DependencyInjection;
+
+use Doctrine\Bundle\MigrationsBundle\DependencyInjection\DoctrineMigrationsExtension;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Sylius\Bundle\CoreBundle\DependencyInjection\SyliusCoreExtension;
+use SyliusLabs\DoctrineMigrationsExtraBundle\DependencyInjection\SyliusLabsDoctrineMigrationsExtraExtension;
+
+final class SyliusCoreExtensionTest extends AbstractExtensionTestCase
+{
+    /**
+     * @test
+     */
+    public function it_autoconfigures_prepending_doctrine_migrations_with_proper_migrations_path_for_test_env(): void
+    {
+        $this->autoConfigurePrependingDoctrineMigrations('test');
+    }
+
+    /**
+     * @test
+     */
+    public function it_autoconfigures_prepending_doctrine_migrations_with_proper_migrations_path_for_test_cached_env(): void
+    {
+        $this->autoConfigurePrependingDoctrineMigrations('test_cached');
+    }
+
+    /**
+     * @test
+     */
+    public function it_autoconfigures_prepending_doctrine_migrations_with_proper_migrations_path_for_dev_env(): void
+    {
+        $this->autoConfigurePrependingDoctrineMigrations('dev');
+    }
+
+    /**
+     * @test
+     */
+    public function it_not_autoconfigures_prepending_doctrine_migrations_with_proper_migrations_path_for_test_env(): void
```

Comment:
```suggestion
    public function it_does_not_autoconfigure_prepending_doctrine_migrations_if_it_is_disabled_for_test_env(): void
```
Here and in other functions